### PR TITLE
[FEAT]Added Foods to the Crafting Menu

### PR DESCRIPTION
**Pull request by RoboTiberius**
I have added the following:

     - A new menu in the Blacksmith menu that says "Food"
     - Added a new item in the "Food" tab called flour
     - Flour costs 0.1 SFF and 3 Wheat, This can be changed later on of course

You can test it by:

     - Collecting at least 0.1SFF and 3 Wheat
     - Go to the Blacksmith menu and click on the last tab called "Foods"
     - Go to the Flour and press Craft
     - Bam! Now you got your very own Flour In SunFlower Land

### CODE REVIEW

File: .idea/.gitignore [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Default ignored files
```
Comment (adamhannigan) [inline .idea/.gitignore:1]: It looks like you have committed some local IDE settings. I would suggest omitting these so others don't pull these down.

File: src/features/blacksmith/components/Crafting.tsx [hunk 2]
```diff
@@ -28,6 +29,10 @@ export const Crafting: React.FC<Props> = ({ onClose }) => {
             <img src={nft} className="h-5 mr-2" />
             <span className="text-sm text-shadow">Items</span>
           </Tab>
+          <Tab isActive={tab === "foods"} onClick={() => setTab("foods")}>
+            <img src={food} className="h-5 mr-2" />
+            <span className="text-sm text-shadow">Foods</span>
```
Comment (adamhannigan) [inline src/features/blacksmith/components/Crafting.tsx:34]:
```suggestion
            <span className="text-sm text-shadow">Food</span>
```

In this case, I believe 'Food' can be used as the plural. It sounds a bit weird using `Foods`

**Review by adamhannigan — CHANGES_REQUESTED** [review]
Nice work! I've run it locally and tested it manually.

I have just left a couple small improvements on the naming side of things.

Thanks for contributing!

File: src/features/blacksmith/components/Crafting.tsx (hunk 2 above)
Comment (RoboTiberius) [reply src/features/blacksmith/components/Crafting.tsx:34]: Okay I'll work on it later today

**Comment by RoboTiberius** [comment]
I have changed everything you have asked for!

**Review by adamhannigan — APPROVED** [review]
LGTM. Nice work 👍

**Comment by RoboTiberius** [comment]
Thanks!

**Comment by RoboTiberius** [comment]
I have fixed it!

**Comment by adamhannigan** [comment]
@RoboTiberius Nice job, welcome to the contributing community 💪

**Pull request merged**